### ISSUE TITLE
Template some blog posts in handlebars

### ISSUE THREAD

**Issue by michaeljonathanblack** [wordstosong/jacobscript#34]
Get a handful of lorem whatever handlebars  files together as example blog posts. We can then add name them with ids that are keyed on the json strips objects in some way. e.g. strip-1234.handlebars

Alternatively, if you'd rather write markdown instead of html, do that, and then we can figure out how we want to render them to the templates before serving them to the client!

**Comment by wordstosong** [comment]
By blog posts, are you referring to comics (since they all have the "bloggy" section underneath)? I'm not planning on doing any non-comic-related posts.

**Comment by michaeljonathanblack** [comment]
Yup, that's why they'd be keyed on the strips (comic strips, strips for short).
